view/src/main/java/org/xcolab/view/pages/proposals/view/proposal/json/ProposalImpactJSONController.java

```diff
@@ -139,12 +145,19 @@ public void proposalImpactDeleteDataSeries(HttpServletRequest request,
             return;
         }
 
-        FocusArea focusArea = OntologyClientUtil.getFocusArea(focusAreaId);
         Proposal proposal = proposalContext.getProposal();
 
-        for (ProposalAttribute proposalAttribute : ProposalAttributeClientUtil
-                .getImpactProposalAttributes(proposal, focusArea)) {
-            proposalContext.getClients().getProposalAttributeClient().deleteProposalAttribute(proposalAttribute.getId_());
+        //TODO: we need to delete *all versions* of these attributes (with the given additionalId)
```

Comment:
Please add JIRA issue number to this TODO.

Reply:
Thanks for noticing, this TODO is completed (I've removed it). This is definitely an advantage of this convention, otherwise this random TODO would have stayed in the code!